.travis.yml

```diff
@@ -8,7 +8,9 @@ cache:
 before_script:
   - npm install -g gulp-cli
 
-script: gulp test-ci
+script:
+  - gulp test-ci
```

Comment:
This follows on from the comment I made about adding **htmlhint** to the `test` gulp task.

`gulp test-ci` currently also runs **StandardJS**. On this though I have no firm opinion, as long as we are consistent.

So either we add **htmlhint** to the `test-ci` task, or on the CI server we run each task individually i.e. standardjs, htmlhint, then unit tests.

My *preference* is the later, but I'm going to let democracy decide this one (every tyrant has to dabble in it if the want to keep the masses happy 😈 )

Reply:
Think I'd prefer separate tasks too.